packages/transformer/src/Algo.ts

```diff
@@ -43,6 +40,26 @@ export function rangesFromRangeAndSkipped(start: number, end: number, skipped: n
   return ranges;
 }
 
+function findRangeContaining(pt: number, inRanges: [number, number][]): number {
+  let begin = 0;
+  let end = inRanges.length - 1;
+  while (end >= begin) {
+    // Instead of Math.floor we could use bitwise operators like ~~ to floor, which seems slightly faster. But I'm not sure of the full effect of that change.
```

Comment:
did you microbenchmark or something? Math.floor is fine and v8 probably optimizes it, and this is not our hot path anyway so I'd just leave it more readable.

Reply:
Yes I ran a few locally with math.floor vs ~~, and ~~ did appear to be a bit faster sometimes 8s vs 6.6s with 100m iterations. Sometimes they were tied at 8s. I agree, I removed that comment.